habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hierarchical_srl_learned_nav.yaml

```diff
@@ -0,0 +1,87 @@
+name: "HierarchicalPolicy"
```

Comment:
similarly, lets call this "hierarchical_srl.yaml" since that is already a PDDL hl and RL skills right?